tests_selenium_waits/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from selenium import webdriver
+
+
+@pytest.fixture(scope="module")
+def driver():
+    chrome_driver = webdriver.Chrome()
+    chrome_driver.maximize_window()
+    yield chrome_driver
+    chrome_driver.implicitly_wait(10)
```

Comment:
adding the implicit wait after yielding the driver doesn't make any sense, because you won't have the wait in the tests